Alberto-todo-list/src/components/Container.jsx

```diff
@@ -1,40 +1,57 @@
-/* Declaracion de dominios*/ 
-import React, { useState } from "react";
-
+import { useState } from "react";
 
 /*Importar los componentes que se van a utilizar en el componente actual
-En este caso se utilizan los componentes principales del Check list para 
+En este caso se utilizan los componentes principales del Check list para
 organizarlos en el container principal
-*/ 
+*/
 import TaskList from "./TaskList";
 import FormTodo from "./FormTodo";
 
-const Container = () => {
-
-  /* Se crea una variable de estado (Hook) en donde iremos guardando todas las actividades 
+const TodoList = () => {
```

Comment:
just a renaming suggestion, instead of `<Container />`, from the App perspective this is , in fact, the `<TodoList/>`,

you can even add an index.js file in your components/ folder exporting just the `<Todolist/>` component, keeping "private" the rest of modules/components/files